optimize/backend/src/main/java/io/camunda/optimize/rest/security/ccsm/CCSMAuthenticationCookieFilter.java

```diff
@@ -123,6 +123,15 @@ protected Object getPreAuthenticatedCredentials(final HttpServletRequest request
         .orElseGet(() -> AuthCookieService.getAuthCookieToken(request).orElse(null));
   }
 
+  private Optional<String> validToken(final String token) {
```

Comment:
❓ Why add this method and its usages? This is adding token verification to places it did not exist previously

Reply:
Yes, that's correct - I've ran into issue without it. The problem is that on token expiration we are removing it from cookies, but still pass it further down to this place for processing (there was no option to replace a token in the request with a refreshed one). So to prevent using the outdated token (which was causing further errors) I've added a token verification here. Thanks for asking this - I wanted to explain this change, but forgot. Maybe there still is some more elegant solution for this, but I didn't manage to find any in a reasonable time.